lib/Conversion/HWToBTOR2/HWToBTOR2.cpp

```diff
@@ -458,21 +464,32 @@ struct ConvertHWToBTOR2Pass
     return width;
   }
 
+  // Emits the btor2 instructions required to define a state transition system
+  // for the given register
+  void finalizeGenRegVisit(Operation *reg, size_t width, Value next,
+                           Value resetVal) {}
+
```

Comment:
I guess this one slipped through accidentally?

Reply:
Yes I forgot to remove that one sorry, the pass got quite large so I miss these sometimes ^^"